tutorial/go-live-checklist.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Checklist
+parent: Go-Live Checklist
+grand_parent: Malibu Tutorial
+nav_order: 01
+---
+
+# {{page.title}}
+
+*This tutorial was contributed by [Deo Kumar](https://www.linkedin.com/in/deo-kumar)*
+
+There are few points that needs to be check before and after Go-Live
+
+## Before Go-live checklist
+
+### Black Knight config
+
+The infra team at Quintype will set up your environment on black knight for you. Once the initial setup is done by infra, you need to add your respective feature configurations in the config file of black night.
+
```

Comment:
I feel the statement should be something like, to make sure if the config in blackknight is set properly(blackknight typo everywhere)

Reply:
To make sure if the config in blackknight is set properly, is this statement needed ?